modules/network/portm/backend/lock.go

```diff
@@ -0,0 +1,46 @@
+package backend
+
+import (
+	"os"
+	"path"
+
+	"github.com/alexflint/go-filemutex"
+)
+
+// FileLock wraps os.File to be used as a lock using flock
+type FileLock struct {
+	f *filemutex.FileMutex
+}
+
+// NewFileLock opens file/dir at path and returns unlocked FileLock object
+func NewFileLock(lockPath string) (*FileLock, error) {
```

Comment:
I think it will be cleaner if this method will return `FileMutex` directly (drop the wrapper). I think the only reason u created this file is the code in the `NewFileLock` method, but u don't need the wrapper struct.

Worst case u define the wrapper as
```go
type FileLock struct {
*filemutex.FileMutex
}
```
this way, u don't need the proxy methods.